content/blog/your-perfect-infrastructure-sucks/index.md

```diff
@@ -1,97 +1,10 @@
 ---
-title: "Your Perfect Infrastructure Sucks"
-date: 2024-12-16T10:43:07Z
-draft: false
-
-meta_desc: |
-    Building systems designed for easy change is more valuable than
-    attempting to create the perfect future-proof architecture.
-
-# The meta_image appears in social-media previews and on the blog home page. A
-# placeholder image representing the recommended format, dimensions and aspect
-# ratio has been provided for you.
-meta_image: meta.png
-
-# At least one author is required. The values in this list correspond with the
-# `id` properties of the team member files at /data/team/team. Create a file for
-# yourself if you don't already have one.
-authors:
-    - simen-a-w-olsen
-
-# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+title: "Your Perfect Infrastructure May Be Not So Perfect"
+date: 2016-12-16T10:43:07Z
+draft: true
 tags:
     - architecture
-    - developer-first-infrastructure
-    - best-practices
-    - cloud-engineering
-    - cloud-deployment
-    - developer-experience-devex
-    - people-ops
-    - application-scalability
-
-
-# The social copy used to promote this post on Twitter and Linkedin. These
-# properties do not actually create the post and have no effect on the
-# generated blog page. They are here strictly for reference.
-
-# Here are some examples of posts we have made in the past for inspiration:
-# https://www.linkedin.com/feed/update/urn:li:activity:7171191945841561601
-# https://www.linkedin.com/feed/update/urn:li:activity:7169021002394296320
-# https://www.linkedin.com/feed/update/urn:li:activity:7155606616455737345
-# https://twitter.com/PulumiCorp/status/1763265391042654623
-# https://twitter.com/PulumiCorp/status/1762900472489185492
-# https://twitter.com/PulumiCorp/status/1755637618631405655
-
-social:
-    twitter: |
-        Perfect architecture is a myth. Simen's take: Make 👏 Change 👏 Cheap 👏
-        
-        Read this candid take on why designing for change beats trying to
-        predict the future - and why your over-engineered system might be holding you back.
-    linkedin: |
-        "I was basically the Leonardo da Vinci of system design... if
-        Leonardo had been really into Kubernetes and had a concerning
-        addiction to coffee." @Simen A. W. Olsen shares how his
-        "future-proof" architecture became a constraint rather
-        than an enabler, and why trying to predict every future need
-        is like bringing a fully packed suitcase to a first date.
-
-
-# See the blogging docs at https://github.com/pulumi/docs/blob/master/BLOGGING.md
-# for details, and please remove these comments before submitting for review.
+authors:
+    - simen-a-w-olsen
+redirect_to: /blog/your-perfect-infrastructure/
```

Comment:
Leaving a redirect from old name, because the build will fail due to leaving a 404 behind.